plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -382,6 +393,8 @@ describe('PersonState.update()', () => {
                 version: 0,
             })
         )
+
+        await delayUntilEventIngested(fetchPersonsRows)
```

Comment:
This might be insufficient depending on tests as some will emit multiple rows to the table.

I'd rather not add this to every test as this is just noise/teardown logic rather anything to do with tests.